src/shared/components/palette/text/write-text-field.component.ts

```diff
@@ -11,6 +11,10 @@ export class WriteTextFieldComponent extends AbstractFieldWriteComponent impleme
   textControl: FormControl;
 
   ngOnInit() {
-    this.textControl = this.registerControl(new FormControl(this.caseField.value)) as FormControl;
+    const notEmpty = this.caseField.value !== null && this.caseField.value !== undefined;
+    this.textControl = this.registerControl(new FormControl(notEmpty ? this.caseField.value : null)) as FormControl;
+    if (this.textControl.disabled) {
```

Comment:
And again:
This takes no account of why the control has been disabled. The conditional show/hide logic will hide and disable fields that shouldn't be shown so that the data from those hidden fields doesn't get submitted. If you've added code to simply enable anything that's disabled - which is what this does - there's every chance the submitted data will be invalid or, at least, that you'll be submitting data for fields that are hidden and shouldn't be submitted.